botocore/waiter.py

```diff
@@ -75,6 +75,12 @@ def wait(self, **kwargs):
     )
 
 
+def is_valid_waiter_error(response):
+    error = response.get('Error')
+    if isinstance(error, dict) and 'Code' in error:
+        return True
+
```

Comment:
Generally, it's best to have boolean functions like this to return explicitly `True` and `False`, instead of `True` and `None` in this case. Could we get an explicit default return value?